src/react-components/src/components/01-atoms/Forms/FormDropdown.tsx

```diff
@@ -0,0 +1,47 @@
+// AF-121 Form Dropdown
+import * as React from "react";
+import bem from "../../../utils/bem";
+import Icon from "../Images/Icons/Icon";
+
+export interface FormDropdownProps {
+  dropdownId: string;
+  blockName?: string;
+  labelText?: string;
+  ariaLabel?: string;
+  disabled?: boolean;
+  options: string[];
+  selectedOption?: string;
+  onSelectBlur: (event: React.FormEvent) => void;
+  onSelectChange: (event: React.FormEvent) => void;
+}
+
+export default function FormDropdown(props: FormDropdownProps) {
+  const { dropdownId, blockName, labelText, options, ariaLabel, disabled, selectedOption, onSelectBlur, onSelectChange } = props;
+  if (!labelText) {
+    if (!ariaLabel) {
```

Comment:
You can do `if (!labelText || !ariaLabel)`

Reply:
Actually, here it says if both are not defined but the error says "Either". So that's a bit confusing.